### DISCUSSION TITLE
MOOSE for plastic forming processes (forging, rolling)

### DISCUSSION THREAD

**Discussion by CinekPoland** [idaholab/moose#23281] [Q&A General] [answered]
Hello everyone. Can the "moose" software effectively simulate plastic forming processes such as forging, rolling? Does the software scale well?

**Comment by GiudGiud** [comment, marked as answer]
Hello

The software scales very well overall, see our publications for this. It will depend on the particular application.

I dont think we have plastic industrial processes implemented, but we are able to model plasticity. It might just be a single step missing, I am ignorant about this

Guillaume
